packages/x-components/tests/e2e/common/common-steps.spec.ts

```diff
@@ -242,6 +242,17 @@ Then(
   }
 );
 
+Then(
+  'search request contains extra parameter {string} with value {string}',
+  (key: string, value: string) => {
+    cy.wait('@interceptedResults')
+      .its('request.body')
+      .then(JSON.parse)
+      .its('extraParams')
+      .should('have.property', key, value === 'default' ? '' : value);
```

Comment:
Is this ternary operator really necessary? I think this was just to handle the sort, but as it is not an extra param it shouldn't be necessary.

Reply:
Fixed